### PR TITLE
build: add runner for LogicTest's bigtest config

### PR DESCRIPTION
**Pull request by jordanlewis**
We used to run this by spinning up a GCE cluster. It'll be simpler to just run it serially on a build agent.

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/13727)

### CODE REVIEW

File: build/teamcity-sqllogictest.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```
Comment (tamird) [inline build/teamcity-sqllogictest.sh]: we use #!/usr/bin/env bash elsewhere

**Review by tamird — APPROVED** [review]

**Comment by tamird** [comment]
We should remove the scripts from cockroach-prod if this is successful (I wonder how long this will take to run).

**Comment by jordanlewis** [comment]
Yes removing the `cockroach-prod` (and terraform configs in `cloud/gce/sql-logic-test`) would be the next step here. My guess is that this will take several hours, but (hopefully) a lot less than 24 hours.

**Comment by jordanlewis** [comment]
Yeah the whole test didn't take long on my machine:

`ok      github.com/cockroachdb/cockroach/pkg/sql        3432.942s`

**Comment by tamird** [comment]
Nice!

**Comment by jordanlewis** [comment]
https://teamcity.cockroachdb.com/viewLog.html?buildId=158899&buildTypeId=Cockroach_SqlLogicTests&tab=buildLog#_state=771 👍

**Pull request merged**